### ISSUE TITLE
Permacultureetc.com broken, videos hosted on Youtube

### ISSUE THREAD

**Issue by raph82** [yt-dlp/yt-dlp#5987]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting a broken site
- [X] I've verified that I'm running yt-dlp version **2023.01.06** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Worldwide (I guess)

### Provide a description that is worded well enough to be understood

Videos on [permacultureetc.com](http://www.permacultureetc.com/) are hosted on Youtube.

yt-dlp fails to download videos from permacultureetc.com with an error message that indicates the Youtube URL (visible below).

yt-dlp has no trouble downloading the same videos with their Youtube URL (also in the verbose output below).
I remember downloads from permacultureetc.com working, with output from the [generic] then the [youtube] extractors (not sure what they're called).

I guess there's an indirection missing now, something like a call from the generic to the youtube extractor.

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['-vU', '-f', 'best[height<=720]/bv[height<=720]+ba', 'http://www.permacultureetc.com/2022/12/comment-greffer-facilement-les-arbres-fruitiers.html']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2023.01.06 [6becd25] (zip)
[debug] Python 3.8.10 (CPython x86_64 64bit) - Linux-5.4.0-136-generic-x86_64-with-glibc2.29 (OpenSSL 1.1.1f  31 Mar 2020, glibc 2.31)
[debug] exe versions: ffmpeg 4.2.7, ffprobe 4.2.7
[debug] Optional libraries: Cryptodome-3.6.1, certifi-2019.11.28, secretstorage-2.3.1, sqlite3-2.6.0
[debug] Proxy map: {}
[debug] Loaded 1760 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Latest version: 2023.01.06, Current version: 2023.01.06
yt-dlp is up to date (2023.01.06)
[generic] Extracting URL: http://www.permacultureetc.com/2022/12/comment-greffer-facilement-les-arbres-fruitiers.html
[generic] comment-greffer-facilement-les-arbres-fruitiers: Downloading webpage
WARNING: [generic] Falling back on generic information extractor
[generic] comment-greffer-facilement-les-arbres-fruitiers: Extracting information
[debug] Looking for embeds
[debug] Identified a Embedly embed
ERROR: No suitable extractor (Embedly) found for URL https://www.youtube.com/watch?v=pfUK_ADTvgY
  File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/usr/local/bin/yt-dlp/__main__.py", line 17, in <module>
    yt_dlp.main()
  File "/usr/local/bin/yt-dlp/yt_dlp/__init__.py", line 970, in main
    _exit(*variadic(_real_main(argv)))
  File "/usr/local/bin/yt-dlp/yt_dlp/__init__.py", line 960, in _real_main
    return ydl.download(all_urls)
  File "/usr/local/bin/yt-dlp/yt_dlp/YoutubeDL.py", line 3358, in download
    self.__download_wrapper(self.extract_info)(
  File "/usr/local/bin/yt-dlp/yt_dlp/YoutubeDL.py", line 3333, in wrapper
    res = func(*args, **kwargs)
  File "/usr/local/bin/yt-dlp/yt_dlp/YoutubeDL.py", line 1491, in extract_info
    return self.__extract_info(url, self.get_info_extractor(key), download, extra_info, process)
  File "/usr/local/bin/yt-dlp/yt_dlp/YoutubeDL.py", line 1502, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/local/bin/yt-dlp/yt_dlp/YoutubeDL.py", line 1599, in __extract_info
    return self.process_ie_result(ie_result, download, extra_info)
  File "/usr/local/bin/yt-dlp/yt_dlp/YoutubeDL.py", line 1678, in process_ie_result
    return self.extract_info(
  File "/usr/local/bin/yt-dlp/yt_dlp/YoutubeDL.py", line 1494, in extract_info
    self.report_error(f'No suitable extractor{format_field(ie_key, None, " (%s)")} found for URL {url}',
  File "/usr/local/bin/yt-dlp/yt_dlp/YoutubeDL.py", line 1012, in report_error
    self.trouble(f'{self._format_err("ERROR:", self.Styles.ERROR)} {message}', *args, **kwargs)
  File "/usr/local/bin/yt-dlp/yt_dlp/YoutubeDL.py", line 941, in trouble
    tb_data = traceback.format_list(traceback.extract_stack())

yt-dlp -vU -f 'best[height<=720]/bv[height<=720]+ba' https://www.youtube.com/watch?v=pfUK_ADTvgY
[debug] Command-line config: ['-vU', '-f', 'best[height<=720]/bv[height<=720]+ba', 'https://www.youtube.com/watch?v=pfUK_ADTvgY']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2023.01.06 [6becd25] (zip)
[debug] Python 3.8.10 (CPython x86_64 64bit) - Linux-5.4.0-136-generic-x86_64-with-glibc2.29 (OpenSSL 1.1.1f  31 Mar 2020, glibc 2.31)
[debug] exe versions: ffmpeg 4.2.7, ffprobe 4.2.7
[debug] Optional libraries: Cryptodome-3.6.1, certifi-2019.11.28, secretstorage-2.3.1, sqlite3-2.6.0
[debug] Proxy map: {}
[debug] Loaded 1760 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Latest version: 2023.01.06, Current version: 2023.01.06
yt-dlp is up to date (2023.01.06)
[youtube] Extracting URL: https://www.youtube.com/watch?v=pfUK_ADTvgY
[youtube] pfUK_ADTvgY: Downloading webpage
[youtube] pfUK_ADTvgY: Downloading android player API JSON
[debug] Sort order given by extractor: quality, res, fps, hdr:12, source, vcodec:vp9.2, channels, acodec, lang, proto
[debug] Formats sorted by: hasvid, ie_pref, quality, res, fps, hdr:12(7), source, vcodec:vp9.2(10), channels, acodec, lang, proto, filesize, fs_approx, tbr, vbr, abr, asr, vext, aext, hasaud, id
[info] pfUK_ADTvgY: Downloading 1 format(s): 22
[debug] Invoking http downloader on "https://rr5---sn-25ge7nsk.googlevideo.com/videoplayback?expire=1673136413&ei=vbS5Y_HUNZ2svdIPq6ih-Ag&ip=2a01%3Acb05%3A3b5%3Abc00%3Ab89%3Aa29%3A330a%3A9077&id=o-AEfeYfTviW0RklGnHIf15dmWQ5F7tA8gRY3O43LJ0wv9&itag=22&source=youtube&requiressl=yes&mh=5Y&mm=31%2C29&mn=sn-25ge7nsk%2Csn-25glenl6&ms=au%2Crdu&mv=m&mvi=5&pl=41&initcwndbps=1240000&spc=zIddbPijIgq8YYemA56eUWlJf8gY9As&vprv=1&svpuc=1&mime=video%2Fmp4&cnr=14&ratebypass=yes&dur=1525.667&lmt=1670664601069638&mt=1673114503&fvip=1&fexp=24007246&c=ANDROID&txp=5432434&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Ccnr%2Cratebypass%2Cdur%2Clmt&sig=AOq0QJ8wRQIhAMnZ4OMMzSV8kCv7wX_uJttr-4s6UrwGhvJD7JfWvwkpAiBXLTi1ihEKYFGBhl7DYm97k8qeYC9DLkpwJ65WfIIe8A%3D%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRAIgAihdYSImSsUPK0qThRfefhuGnyFUktw0xfjh9a3gMRoCIG46BcvbXnjg4oh9-xVz7NWTu0M5L4gcXTWO-I2UCBuT"
[download] Destination: Comment greffer facilement les arbres fruitiers ？ (mois par mois) [pfUK_ADTvgY].mp4
[download]   0.6% of  163.69MiB at  998.02KiB/s ETA 02:46^C
ERROR: Interrupted by user
```

**Comment by raph82** [comment]
Wow, impressive time-to-fix.

It looks like I spend more time on _carefully_ writing my first bug report for yt-dlp than @pukkandan took coding the fix! I'm so glad.

Congrats! And great issue templates!

**Comment by raph82** [comment]
Fix confirmed in 2023.02.17
